tests/testthat/testSel3D.R

```diff
@@ -0,0 +1,55 @@
+# testSel3D.R
+#
+#
+# Structure your tests according to the three principles expressed below:
+#    test_that("parameter errors are correctly handled", { ... })
+#    test_that("a sane input gives an expected output", { ... })
+#    test_that("a corrupt input does not lead to corrupted output", { ...})
+
+
+# ==== BEGIN SETUP AND PREPARE =================================================
+#OLOG <- as.character(getOption("rete.logfile"))   # save original logfile name
+#logFileName(fPath = tempdir(), setOption = TRUE)  # make tempdir() the log dir
+#NL <- .PlatformLineBreak()
+# ==== END SETUP AND PREPARE ===================================================
+
+
+
+
+#test_that("parameter errors are correctly handled", {
+  # Try missing input sequence
+  # Try extraneous parameters
+  # Note: we're not to attempt to avoid all kinds of abuse from users
```

Comment:
The following link has information about proper documentation. 
http://r-pkgs.had.co.nz/man.html
In particular, 
"@param name description describes the function’s inputs or parameters. The description should provide a succinct summary of the type of the parameter (e.g., string, numeric vector) and, if not obvious from the name, what the parameter does."